CMakeLists.txt

```diff
@@ -282,6 +282,10 @@ if(ACTS_SETUP_BOOST)
         find_package(Boost ${_acts_boost_version} REQUIRED COMPONENTS)
     endif()
 
+    set(_severity WARNING)
+    if(ACTS_BUILD_EXAMPLES)
+        set(_severity FATAL_ERROR)
+    endif()
```

Comment:
nitpick: `_severity` seems a bit generic

Alternatively we could also put the error into the `Examples/CMakeLists.txt`

Reply:
If I put it in `Examples/CMakeLists.txt` I'd have to duplicate the version check for Boost, which I'd like to avoid.
I changed the `_severity` variable to `_boost_version_severity`, do you think that's better?